tests/kani/Intrinsics/ArithOffset/offset_u8_ok.rs

```diff
@@ -18,7 +18,7 @@ fn test_arith_offset() {
 
         // This is okay because it's one byte past the object,
```

Comment:
We should probably update the comment in these tests and use a `_x` to show that this can't really be de-referenced.

Reply:
Done